sdk/containerservice/Azure.ResourceManager.ContainerService/src/autorest.md

```diff
@@ -16,18 +16,48 @@ skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
-list-exception:
-- /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/locations/{location}/osOptions/default
+request-path-to-singleton-resource:
+  /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/locations/{location}/osOptions/default: osOptions/default
 
 rename-mapping:
   ManagedClusterPodIdentityProvisioningError.error: 'ErrorDetail'
+  Code: ManagedClusterStateCode
+  Format: KubeConfigFormat
```

Comment:
Not sure if this name will be dup with other Container services?

Reply:
It could be, will revisit duplicate names after I worked on all 3 conatiner RPs.